op-guide/ansible-deployment.md

```diff
@@ -119,29 +119,34 @@ The key's randomart image is:
 | tidb-ansible 分支 | TiDB 版本 | 备注 |
 | ---------------- | --------- | --- |
 | release-2.0 | 2.0 版本 | 最新稳定版本，可用于生产环境。 |
+| release-2.1 | 2.1 版本 | 最新 2.1 RC 版本，不建议用于生产环境。 |
 | master | master 版本 | 包含最新特性，每日更新。 |
 
 使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应分支，默认的文件夹名称为 `tidb-ansible`。
 
-下载 2.0 版本：
+- 下载 2.0 版本：
 
-```
-$ git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
-```
+	```
```

Comment:
Tab space is not allowed in docs/docs-cn, which causes CircleCI failure. Use whitespaces instead.